---
title: El operando 'Throw' debe derivar de 'System.Exception'
ms.date: 07/20/2015
f1_keywords:
- vbc30665
- bc30665
helpviewer_keywords:
- BC30665
ms.assetid: 7c228087-39ea-4b30-a410-6ba711e67e5e
ms.openlocfilehash: 9fb5b4e1193c120db70088b1d7c96177b8ccea06
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: HT
ms.contentlocale: es-ES
ms.lasthandoff: 04/23/2019
ms.locfileid: "61666136"
---
# <a name="throw-operand-must-derive-from-systemexception"></a>El operando 'Throw' debe derivar de 'System.Exception'
El argumento proporcionado a `Throw` debe ser una instancia de `System.Exception` o una instancia de una clase derivada de `System.Exception`.  
  
 **Identificador de error:** BC30665  
  
## <a name="to-correct-this-error"></a>Para corregir este error  
  
- Use un argumento derivado de `System.Exception`, como en el ejemplo siguiente.  
  
    ```vb
    Throw New System.Exception("This is an error.")  
    ```  
  
## <a name="see-also"></a>Vea también

- [Throw (instrucción)](../../visual-basic/language-reference/statements/throw-statement.md)
- [Try...Catch...Finally (instrucción)](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
- <xref:System.Exception?displayProperty=nameWithType>
- [Controlar y generar excepciones en .NET](../../standard/exceptions/index.md)
